cmd/lit-af/lit-af.go

```diff
@@ -163,6 +163,13 @@ func (lc *litAfClient) litAfSetup(conf litAfConfig) error {
 		if err != nil {
 			logging.Fatal(err.Error())
 		}
+	} else if len(adr) == 0 {
+		// so the user didn't provide us with an address to connect to
+		// we need to connect to the locally running lit-af instance
+		lc.RPCClient, err = litrpc.NewLocalLndcRpcClientWithHomeDirAndPort(defaultDir, port)
```

Comment:
Can we make a way to specify a homedir too?  Because without that it kinda breaks my workflow.

Reply:
hm.. yeah, I think that would be possible, was possible earlier as well iirc. Will push an update.

Reply:
This seems to work right now (`./lit-af --dir=blah`), was this the use case you were looking at?